packages/compass/src/app/styles/reset.less

```diff
@@ -0,0 +1,69 @@
+body {
```

Comment:
Is this what LG recommend / uses in the showcase?

Reply:
[I asked leafygreen](https://mongodb.slack.com/archives/CFFA5BS79/p1645111726152939),  they just require one rool. This reset css is more aggressive than we need I think. Ultimately it's up to us, going to see if there's anything we should remove. We can also tweak things overtime and reduce what we have here.